### ISSUE TITLE
Blank scree when opening Anki

### ISSUE THREAD

**Issue by bsauce222** [Arthur-Milchior/anki-enhance-main-window#79]
Whenever I have the enhance main window add on enabled it gives me a blank screen. I have to close the app, hold shift and then disable the add on.
I'd love a fix, I'm a bid fan of the add on and I have 0 knowledge of coding.
Appreciate any help.

**Comment by Arthur-Milchior** [comment]
Sadly, I have never been able to solve this one. Usually, it seems to be solved this way:
* sync your collection
* create a new profile in anki (ctrl+P to see profile manager, then new)
* sync this new profile using the same anki account
You'll see exactly the same collection, but sometime it solve the add-on problem.

I never understood why.

**Comment by cjdduarte** [comment]
This has already happened to me. It was conflict with another addon. When I found out, it worked again.
But it's been so long that I don't remember what it was. I recommend disabling others until you find out.

**Comment by Arthur-Milchior** [comment]
I don't recall there was conflict. That is really something I would like to know

**Comment by bsauce222** [comment]
I just tried individual add ons to see if there was a conflict and there wasn't. I currently syncing everything to a separate profile to see if that works. I'll update if there is any progress and thank you all for your help!

**Comment by bsauce222** [comment]
Unfortunately making a new profile did not help as well and I have to leave it disabled

**Comment by Arthur-Milchior** [comment]
May I ask how many deck you have displayed (if you close a deck and don't see its subdecks, don't count the subdecks). Which computer/version of anki you use

**Comment by bsauce222** [comment]
Absolutely! If I close everything, I have 13 decks and I currently use 2.1.19. I'm also using it on Mac Catalina 10.15.3.

**Comment by Arthur-Milchior** [comment]
Thank you.
I don't have a mac to test. But 13 decks is not a lot. 19 is last version. So there is nothig I can imagine. Really sorry

**Comment by bsauce222** [comment]
Its all good! Thank you for your help at least. If anything changes or if you think of anything else let me know! I did enjoy the add on!

**Comment by feldspar9** [comment]
I am having this same issue.
Please let me know if you need any more information. I loved this add-on - it had a unique feature set but I had to disable it for the time being.

**Comment by feldspar9** [comment]
It started working today again randomly.
Don't know why.
Only thing I noticed is I went to addons and check for updates and it updates the 3 Hoochie addons (mama, papa, baby) and restarted.

I don't know if there is any connection between that and it now working.

**Comment by lovac42** [comment]
It should not related. No change has been made to the hoochie addons other than making it look more pretty and hoochie.

**Comment by radioinahole** [comment]
Yeah, basically if you open a deck that has too many subdecks- the addon won't work

So if I have most decks unexpanded- the addon works but if I open my pharma deck or Sketchy deck (which have a lot of subdecks)- I see a blank screen basically

**Comment by radioinahole** [comment]
Basically, I just have to make sure I don't expand too many decks at the same time, then it's fine

Thank you for this great addon btw <3

**Comment by aceserani** [comment]
Hi, just wanted to add to what @radioinahole mentioned on Jan 6. Too many subdecks open all of a sudden seemed to make the addon not work, unless I toggled off the 'More Deck Stats and Time Left' addon as well (which adds some lines to the bottom of the main screen, so maybe that's a related issue).

However, now that I collapsed my subdecks, I can have both on at the same time. Thanks all! Really love this addon, thanks for your work

**Comment by cjdduarte** [comment]
> Hi, just wanted to add to what @radioinahole mentioned on Jan 6. Too many subdecks open all of a sudden seemed to make the main screen go blank, unless I toggled off the 'More Deck Stats and Time Left' addon as well (which adds some lines to the bottom of the main screen, so maybe that's a related issue).
> 
> However, now that I collapsed my subdecks, I can have both addons toggled on at the same time. Thanks all! Really love this addon, thanks for your work

The addon "More Deck Stats and Time Left" is mine, but I never had a problem with it. I've been using the 2 addons for more than 2 years. It must be another one.

**Comment by aceserani** [comment]
> > Hi, just wanted to add to what @radioinahole mentioned on Jan 6. Too many subdecks open all of a sudden seemed to make the main screen go blank, unless I toggled off the 'More Deck Stats and Time Left' addon as well (which adds some lines to the bottom of the main screen, so maybe that's a related issue).
> > However, now that I collapsed my subdecks, I can have both addons toggled on at the same time. Thanks all! Really love this addon, thanks for your work
> 
> The addon "More Deck Stats and Time Left" is mine, but I never had a problem with it. I've been using the 2 addons for more than 2 years. It must be another one.

Hey! Sorry, didn't mean to imply the problem was with your addon. I think radiohole is right that it has something to do with having too many subdecks open, or perhaps, I think, with having too many lines of text in the main window.

In my case, the total lines of decks+subdecks in the main windows right now, with Enhance Main Window on but More Deck Stats and Time Left off, is 143 (I know that's not normal or advised, it's just convenient for some decks I'm messing around with right now). If I toggle on More Deck Stats, or another addon that adds lines/text to the main window like Review Heatmap, the main window goes blank.

Alternatively, if I collapse one of those subdecks and bring the line total to 123, I can have EMW, More Decks Stats, and Reivew Heatmap toggled on with no issues on the main window.

I didn't mean to point out your addon specifically and imply it was the one causing issues, I just used it as an example of an addon that adds lines or text to the main window. My inexpert guess was that if I had enough addons that added lines and text to the main window toggled on, any such combination would cause the window to go blank.

I have zero knowledge in coding or Anki addon writing so I'm sorry if I'm missing some context, just wanted to report what was going on for me in case it was useful. Thank you all for your great addons!

**Comment by cjdduarte** [comment]
OK. I understand your argument.

**Comment by sanatv2s** [comment]
This issue is still there. Just installed Anki 2.1.65 Qt5 using the installation script from the website. 
I tried using shift and clicking on anki to skip sync, tried creating new profile as well and disabling hardware acceleration but it did not work. I am using Ubuntu 22.04 LTS.
